x-pack/plugins/security_solution/public/common/components/alert_count_by_status/use_alert_count_by_rule_by_status.ts

```diff
@@ -202,7 +203,7 @@ const parseAlertCountByRuleItems = (
   return buckets.map<AlertCountByRuleByStatusItem>((bucket) => {
     const uuid = bucket.ruleUuid.hits?.hits[0]?._source['kibana.alert.rule.uuid'] || '';
     return {
-      ruleName: bucket.key,
+      ruleName: isArray(bucket.key) ? bucket.key[0] : bucket.key,
```

Comment:
Not necessary, but it may be useful to utilize this common helper here and all the other places where you do the isArray check. 

https://github.com/elastic/kibana/blob/c2e538189e14698080eb1d14b98f6671b3a32186/x-pack/plugins/security_solution/common/endpoint/models/ecs_safety_helpers.ts#LL28